packages/dds/tree/src/edit-manager/editManager.ts

```diff
@@ -185,8 +210,9 @@ export class EditManager<TChangeset, TChangeFamily extends ChangeFamily<any, TCh
             return;
         }
         const newBranchChanges: Commit<TChangeset>[] = [];
-        const inverses: TChangeset[] = [];
+        const inverses: ChangeWithMetadata<TChangeset>[] = [];
 
+        let nextTempRevision = -1;
         for (const commit of branch.localChanges) {
```

Comment:
I think we want to filter out references to branch changes on the `commit.changeset` (before rebasing it).
My thinking is that...
a. We want to avoid mixing up references to different versions of the same local change.
b. We should not need those references because we now know that they are to changes that will not end up on the trunk.

Reply:
Giving up on filtering for now.